org.apache.zeppelin.interpreter.remote.RemoteInterpreterServerTest$ShutdownRun.run : r1 = $r3 >>>>>>>> org.apache.zeppelin.interpreter.remote.RemoteInterpreterServerTest$ShutdownRun.run : return
org.apache.zeppelin.interpreter.remote.RemoteInterpreterServerTest$ShutdownRun.run : $r2 = r0.<org.apache.zeppelin.interpreter.remote.RemoteInterpreterServerTest$ShutdownRun: org.apache.zeppelin.interpreter.remote.RemoteInterpreterServer serv> >>>>>>>> org.apache.zeppelin.interpreter.remote.RemoteInterpreterServerTest$ShutdownRun.run : virtualinvoke $r2.<org.apache.zeppelin.interpreter.remote.RemoteInterpreterServer: void shutdown()>()
org.apache.zeppelin.interpreter.remote.RemoteInterpreterServerTest$ShutdownRun.run : $r2 = r0.<org.apache.zeppelin.interpreter.remote.RemoteInterpreterServerTest$ShutdownRun: org.apache.zeppelin.interpreter.remote.RemoteInterpreterServer serv> >>>>>>>> org.apache.zeppelin.interpreter.remote.RemoteInterpreterServerTest$ShutdownRun.run : $r3 := @caughtexception
org.apache.zeppelin.interpreter.remote.RemoteInterpreterServerTest$ShutdownRun.run : virtualinvoke $r2.<org.apache.zeppelin.interpreter.remote.RemoteInterpreterServer: void shutdown()>() >>>>>>>> org.apache.zeppelin.interpreter.remote.RemoteInterpreterServerTest$ShutdownRun.run : goto [?= return]
org.apache.zeppelin.interpreter.remote.RemoteInterpreterServerTest$ShutdownRun.run : virtualinvoke $r2.<org.apache.zeppelin.interpreter.remote.RemoteInterpreterServer: void shutdown()>() >>>>>>>> org.apache.zeppelin.interpreter.remote.RemoteInterpreterServerTest$ShutdownRun.run : $r3 := @caughtexception
org.apache.zeppelin.interpreter.remote.RemoteInterpreterServerTest$ShutdownRun.run : goto [?= return] >>>>>>>> org.apache.zeppelin.interpreter.remote.RemoteInterpreterServerTest$ShutdownRun.run : return
